test/effects003/hangman.idr

```diff
@@ -223,3 +223,7 @@ were caught by the type checker:
 
 main : IO ()
 main = run runGame
+
+-- Local Variables:
+-- idris-load-packages: ("effects")
```

Comment:
I did. It lets the code work interactively in idris-mode when debugging test failures.